test/e2e-common.sh

```diff
@@ -147,6 +147,12 @@ function install_knative_eventing() {
       -f "${EVENTING_CORE_NAME}" || return 1
     UNINSTALL_LIST+=( "${EVENTING_CORE_NAME}" )
 
+    local EVENTING_TLS_NAME=${TMP_DIR}/${EVENTING_TLS_YAML##*/}
+    sed "s/namespace: ${KNATIVE_DEFAULT_NAMESPACE}/namespace: ${SYSTEM_NAMESPACE}/g" ${EVENTING_TLS_YAML} > ${EVENTING_TLS_NAME}
```

Comment:
As some of the TLS components use a hard coded cluster domain suffix, could replace `cluster.local` with the value from CLUSTER_SUFFIX too? (something like the following)

```
    if [[ ! -z "${CLUSTER_SUFFIX}" ]]; then
      # Replace cluster.local with the set CLUSTER_SUFFIX
      sed -i "s/cluster.local/${CLUSTER_SUFFIX}/g" ${EVENTING_TLS_NAME}
    fi
```

Reply:
I'd do that in a follow up